src/js/utils/html-utils.js

```diff
@@ -108,7 +90,7 @@
         };
 
         /**
-         * Returns a function that renders an Underscore template as an HTML snippet.
+         * Returns a function that renders an Underscore template as an HTML string.
```

Comment:
Maybe just drop the word "HTML"?  It really all depends on what is done in the template.